### PR TITLE
Add Error Handling For app config errors

### LINKED ISSUE #70: Missing configuration handling
When there is no `App.config` present, the configuration file does not get copied into the build directory. Because it is missing, Miunie will throw an exception.

It would be better if Miunie could output something like:
```
There is no configuration file present.
See [link-to-the-right-README-section].
```

> :bulb: We could also add a feature where you can have the config generated while Miunie asks for your bot token.

### PR DESCRIPTION
**Pull request by DraxCodes**
Fixes #70 

## Additons

* Adds error handling for the app.config.
* Allows users to enter their token if one doesn't exist

## Co-Authors

Co-authored-by: @petrspelos

### CODE REVIEW

File: src/Miunie.Configuration/ConigurationFileEditor.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            file = ConfigurationManager.OpenExeConfiguration(ConfigurationUserLevel.None);
+        }
+
+        internal void Save()
+        {
+            file.Save(ConfigurationSaveMode.Modified);
+            ConfigurationManager.RefreshSection(file.AppSettings.SectionInformation.Name);
+        }
+
+        internal void WriteSetting(string key, string value)
```
Comment (svr333) [inline src/Miunie.Configuration/ConigurationFileEditor.cs]: What if we make it compacter by doing:

```
internal void WriteSetting(string key, string value)
        {
            if (SettingExists(key)) { UpdateSetting(key, value); }
            else { CreateSetting(key, value); }
```

File: src/Miunie.Configuration/ConfigManager.cs [hunk 2]
```diff
@@ -1,10 +1,50 @@
...
+        public string GetValueFor(string key)
+        {
+            var value = ConfigurationManager.AppSettings[key];
+
+            if (value is null)
+            {
+                var editor = new ConfigurationFileEditor();
+
+                Console.ForegroundColor = ConsoleColor.Red;
+                Console.WriteLine("Configuration Error:");
```
Comment (svr333) [inline src/Miunie.Configuration/ConfigManager.cs]: Shouldn't we filter out this hard-coding?

Comment (petrspelos) [reply src/Miunie.Configuration/ConfigManager.cs]: This is an edge case so not in this case.

File: src/Miunie.Configuration/ConigurationFileEditor.cs (hunk 1 above)
Comment (petrspelos) [reply src/Miunie.Configuration/ConigurationFileEditor.cs]: According to the style guide we tend to follow, you shouldn't collapse if statements that have an else branch.

File: src/Miunie.Configuration/ConfigManager.cs (hunk 2 above)
Comment (svr333) [reply src/Miunie.Configuration/ConfigManager.cs]: Okay, was just wondering.

File: src/Miunie.Configuration/ConigurationFileEditor.cs (hunk 1 above)
Comment (svr333) [reply src/Miunie.Configuration/ConigurationFileEditor.cs]: Right. Sorry about that.

**Pull request merged**